docs/MEMBERSHIP.md

```diff
@@ -6,67 +6,167 @@
+  * Attending community events
+  * Talking about the project on social media, blogs, and talks
+
+### Contributor
+
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+  * Following the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+  * Following the project Contributing Guide (to be updated)
+* Requirements (one or several of the below):
+  * Reports, and sometimes resolves issues
+  * Occasionally submits PRs
+  * Contributes to the documentation
+  * Regularly shows up at meetings, takes notes
+  * Answers questions from other community members
+  * Submits feedback on issues and PRs
+  * Tests releases and patches and submits reviews
+  * Runs or helps run events
+  * Promotes the project in public
+  * Helps run the project infrastructure
+* Privileges:
+  * Invitations to Contributor events
+  * Eligible to become a Reviewer or Approver
 
 ### Reviewer
 
-Reviewers are able to review code for quality and correctness on the Secrets Store CSI Driver. They are knowledgeable about the codebase and software engineering principles.
-
-**Defined by:** reviewers entry in the [OWNERS](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/v0.0.22/OWNERS#L5) file
+Description: Reviewers are Contributors who are able to review code for quality and correctness on the Secrets Store CSI Driver. They are knowledgeable about the codebase and software engineering principles. A Reviewer has the rights, responsiblities, and requirements of a Contributor, plus:
+
+* Responsibilities include:
+  * Running tests for PRs from members who aren’t part of Kubernetes org
+    * This involves running ok-to-test after assessing the PR briefly 
+  * Project quality control via code reviews
+    * Focusing on code quality and correctness, including testing and factoring
+    * May also review for more holistic issues, but not a requirement
+  * Responding to review requests as per [community expectations](https://github.com/kubernetes/community/blob/master/contributors/guide/expectations.md)
+  * Assigned PRs to review
+  * Assigned test bugs
+
+* Requirements (one or several of the below):
+  * Reviewers entry in the [OWNERS](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/v0.0.22/OWNERS#L5) file
+  * Membership in kubernetes-sigs org
+  * Reviewer for at least 5 PRs to the codebase
+  * Reviewed or merged at least 10 substantial PRs to the codebase
+  * Knowledgeable about the codebase
+  * Sponsored by a subproject approver
+    * With no objections from other approvers
+    * Done through PR to update the OWNERS file
+  * May either self-nominate, be nominated by an approver in this subproject
+
+* Privileges:
+  * Invitations to Contributor events
+  * Eligible to become a reviewer or approver
```

Comment:
This is in the `Reviewer` section. Should we update this to `Eligible to become an approver`?

Reply:
May lgtm code contributions